autoscale/src/main/java/com/sequenceiq/periscope/monitor/handler/FreeIpaCommunicator.java

```diff
@@ -32,18 +31,8 @@ public FreeIpaCommunicator(UserV1Endpoint userV1Endpoint) {
     @Retryable(value = Exception.class, maxAttempts = 5, backoff = @Backoff(delay = 10000))
     public SyncOperationStatus synchronizeAllUsers(SynchronizeAllUsersRequest request) {
         LOGGER.info("Invoking freeIpa user sync request: {}", request);
-        String envCrn = request.getEnvironments().iterator().next();
-        SyncOperationStatus lastSyncStatus = ThreadBasedUserCrnProvider.doAsInternalActor(
+        return ThreadBasedUserCrnProvider.doAsInternalActor(
```

Comment:
Why this change ? (in the current patch)

Reply:
Added this change because the IPA service throws an error when user sync is invoked for the first time in an env